tmt/steps/provision/testcloud.py

```diff
@@ -788,8 +782,8 @@ def prepare_config(self) -> None:
         self.config.DOWNLOAD_PROGRESS_VERBOSE = False
 
         # Configure to tmt's storage directories
-        self.config.DATA_DIR = TESTCLOUD_DATA
-        self.config.STORE_DIR = TESTCLOUD_IMAGES
+        self.config.DATA_DIR = effective_workdir_root(self.workdir_root) / 'testcloud'
```

Comment:
Shouldn't this be just `self.workdir_root`, without the `effective_workdir_root()` call?